synfig-studio/src/synfigapp/canvasinterface.cpp

```diff
@@ -985,74 +998,98 @@ CanvasInterface::import_sequence(
 			}
 			
 			String short_filename = CanvasFileNaming::make_short_filename(get_canvas()->get_file_name(), filename);
-			
+
 			try {
 				layer = add_layer_to("Import",get_canvas());
 				int w, h;
 				if (!layer)
 					throw int();
 				if (!layer->set_param("filename", ValueBase(short_filename)))
 					throw int();
+
+
+				curr_layer= modules::lyr_std::Import::Handle::cast_dynamic(layer);
+				cur_surface=cur_lock.get_handle();
+
+				//assuming is_same_as to be the pseudo function to compare two bitmap layers
+				if(c2!=std::next(filenames.end(), -1) && !first_time && cur_surface->compare_with(prev_surface) && remove_dups){
```

Comment:
the order matters for efficiency:
`remove_dups`should be the first operand. If it is false, the following operands would not even evaluated :)

Reply:
Okay! I will make the changes! I was focused on getting it to work first :sweat_smile: